source/track_thick.f90

```diff
@@ -1886,317 +1886,324 @@ subroutine synuthck
   use mod_common_da
   implicit none
   integer ih1,ih2,j,kz1,l
-  real(kind=fPrec) fokm
+  real(kind=fPrec) fokm,fok,fok1,rho,si,co,sm1,sm2,sm3,sm12,sm23,as3,as4,as6,g,gl,rhoc,siq,aek,hi,  &
+    fi,hi1,hp,hm,hc,hs,wf,afok,wfa,wfhi,rhoi
+
   save
-!---------------------------------------  SUBROUTINE 'ENVARS' IN-LINE
+
 #ifdef CR
   sythckcr=.true.
 #endif
-  do 10 j=1,napx
-    dpd(j)=one+dpsv(j)
-    dpsq(j)=sqrt(dpd(j))
-10 continue
+  do j=1,napx
+    dpd(j)  = one+dpsv(j)
+    dpsq(j) = sqrt(dpd(j))
+  end do
+
   do 160 l=1,il
     if(abs(el(l)).le.pieni) goto 160
     kz1=kz(l)+1
-!       goto(20,40,80,60,40,60,100,100,140),kz1
-!       goto 160
-!Eric
 !-----------------------------------------------------------------------
 !  DRIFTLENGTH
 !-----------------------------------------------------------------------
-    if (kz1.eq.1) then
+    if(kz1 == 1) then
       goto 20
 !-----------------------------------------------------------------------
 !  RECTANGULAR MAGNET
 !  HORIZONTAL
```

Comment:
See comments about comments in the previous file. I won't repeat this further...